dbt/include/athena/macros/materializations/snapshots/snapshot.sql

```diff
@@ -248,21 +248,17 @@
   {%- set unique_key = config.get('unique_key') %}
   {%- set file_format = config.get('file_format', 'parquet') -%}
 
+  {{ log('Checking if target table exists') }}
   {% set target_relation_exists, target_relation = get_or_create_relation(
-          database=none,
+          database=model.database,
           schema=model.schema,
           identifier=target_table,
           type='table') -%}
 
-
   {% if not adapter.check_schema_exists(model.database, model.schema) %}
     {% do create_schema(model.database, model.schema) %}
   {% endif %}
 
-  {%- if not target_relation.is_table -%}
```

Comment:
This is not required as athena doesn't support views and we can eliminate checking this. Feel free to let me know if this is irrelavant and I'll remove it.

Reply:
Agreed 👍🏻